src/components/addBook.js

```diff
@@ -30,16 +36,18 @@ const AddBook = () => {
         submitBook(title, category);
       }}
     >
+      <h2>Add new book</h2>
```

Comment:
- [ ] According to the design, there should be a styled `hr` element between the book form and the booklist. please add it.